doc/api/modules.md

```diff
@@ -905,15 +905,35 @@ by the [module wrapper][]. To access it, require the `Module` module:
 const builtin = require('module').builtinModules;
 ```
 
+### module.createRequire(filename)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `filename` {string|URL} Filename to be used to construct the require
+  function. Must be an absolute path.
```

Comment:
Can we call this `path` in the documentation if it doesn’t have to refer to an actual file? (That’s also an issue that `createRequireFromPath()` has, I guess.)

Reply:
I updated the docs to be more specific, does that work better? If not please feel lmk what else to change or feel free to push a commit

Reply:
I was thinking about the parameter name itself – `filename` doesn’t seem accurate if it can also be a directory (which too is a kind of file, but that’s not always intuitive), and if we don’t currently enforce it (aka add a `stat` call). `path` would be plainer and seems more accurate to me.

Reply:
I can change it to `filepath`, at least that would be more accurate... thoughts?

Reply:
I guess? My issue was more with the `file` bit than the `name` bit :smile: But it’s also really not that important

Reply:
I'm struggling figuring out the best name. The argument can be a file URL string, a file URL object, and an absolute path string. I'm concerned path is not inclusive of URL, but maybe I'm over thinking it.

sigh